apiary.apib

```diff
@@ -321,48 +309,64 @@ curl http://140.116.247.120:5000/participant -X GET -H "Content-Type: applicatio
 </pre>
 
 + Request (application/json)
-
     + Headers
 
             API-version: 0.1
             Authorization: Bearer OXBiogjCfdKb7m9CEOPhpbjjv0n5OtfltzgMJbSxbP7jVFY8XUNzHqkGidDumQSR - Authentication Bearer Token
 
 + Response 200 (application/json)
-
     + Attributes (array, fixed-type)
         + (object)
             + id: `7e54877b-3743-4d20-bbfc-5b4deb131116` - User's UUID
-            + name: BEMS1
+            + name: `BEMS1`
 
-## Electricity History Usage [/power_info]
+## Power Info [/power_info]
 
-Retuen Electricity History Usage.
+Return Electricity History Usage.
 
-### Electricity History Usage [GET /power_info{?time}]
+### PowerData (Data Table Mode) [GET /power_info{?per_page}{?page}{?time}]
 <pre>
-curl http://140.116.247.120:5000/power_info -X GET -H "Content-Type: application/json" -H "Authorization: Bearer OXBiogjCfdKb7m9CEOPhpbjjv0n5OtfltzgMJbSxbP7jVFY8XUNzHqkGidDumQSR"
-curl http://140.116.247.120:5000/power_info?time=2019-10-19 -X GET -H "Content-Type: application/json" -H "Authorization: Bearer OXBiogjCfdKb7m9CEOPhpbjjv0n5OtfltzgMJbSxbP7jVFY8XUNzHqkGidDumQSR"
+curl http://140.116.247.120:5000/power_info?per_page=10&page=1&time=2020/02/14 -X GET -H "Content-Type: application/json" -H "Authorization: Bearer OXBiogjCfdKb7m9CEOPhpbjjv0n5OtfltzgMJbSxbP7jVFY8XUNzHqkGidDumQSR"
 </pre>
 
 + Parameters
-    + time: `2019-10-19` (optional) - Query date. Format: `YYYY-mm-DD`
-        + Default: System date
+    + per_page: `10` (number, optional) - Power data per page.
+    + page: `1` (number, optional) - Power data at page.
```

Comment:
If there's no setting of `per_page` or `page`, return data will be empty.
For `PowerData Data Table Mode`, these parameters are both required.

Reply:
Noted.

Reply:
Noted.
Will be replace by 'find and replace'